lib/babel-preset.js

```diff
@@ -0,0 +1,21 @@
+exports.buildPreset = buildPreset;
+exports.default = buildPreset();
+Object.defineProperty(exports, '__esModule', { value: true });
+
+function buildPreset(options) {
+  const minNode = (options && options.minNode) || '4.3';
+
+  return {
+    presets: [['env', {
+      useBuiltIns: true,
+      targets: {
+        node: minNode,
+      },
+    }]],
+    plugins: [
+      ['transform-object-rest-spread', { useBuiltIns: true }],
+      ['transform-class-properties', { spec: true }],
+      'transform-export-extensions',
```

Comment:
This is at `stage-1` @tc39. I personally doubt it will every land in the language standard, but in general all `proposals < stage-3`(vendor interest, first vm implementations) shouldn't be in production code bases at all 😛